doc/modules/ROOT/pages/index.adoc

```diff
@@ -128,7 +456,51 @@ image::ss5.5-create-instance.png[]
 
 As the last step, select your https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[key pair]
 for the instances and that's it. You can launch instances now.
+--
+
+{tab-header-CLI}::
++
+--
+Using the id's and the names you set above, now run two EC2 instances. In addition to the steps so far, you need to have
+a https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[key pair] and pass it via `--key-name` flag.
+The command below will start two Amazon Linux machines with type t2.micro and assign public IP addresses to each.
+Also a tag named "cluster-tag" with "guide-ec2-cluster" value will be assigned to the instances:
 
+[source, shell]
+----
+$ aws ec2 run-instances --image-id ami-04d29b6f966df1537 \
```

Comment:
Will this AMI ID always work? I tried and got this exception:
```
The image id '[ami-04d29b6f966df1537]' does not exist
```

Maybe the AMI IDs are different for different regions. I try it on `eu-central-1`.

Reply:
I just noticed that they are region specific. I'll use parameters like `resolve:ssm:/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2` instead.